.github/workflows/GnuTests.yml

```diff
@@ -217,6 +217,9 @@ jobs:
       run: |
         ## Compare test failures VS reference
         have_new_failures=""
+        ## Collect if we detect any non-intermittent failures
+        ## this will be used by util/compare_gnu_result.py
+        have_non_intermittent_failures=""
```

Comment:
Currently `have_non_intermittent_failures` is identical to `have_new_failures` and so either there is a bug somewhere or the variable is not needed.